### PR TITLE
USDZExporter: Anchoring support

### LINKED ISSUE #21995: USDZ exporter with target wall placement
**Is your feature request related to a problem? Please describe.**

When exporting to USDZ currently there is the possibility missing to define the tracking target for AR view (AR Quick-look)

**Describe the solution you'd like**

I'm thinking of a way to make the exporter configurable so that it accepts the different tracking targets supported by Apple AR Quick-look

**Additional context**
Reading through the schema definition here: https://developer.apple.com/documentation/arkit/usdz_schemas_for_ar 
It is possible to encode into the USDZ which tracking target should be used. Hence, I've extended the USDZExporter to support configuration for targets. For now, wall target is integrated but more can be added in the future.

Please see this branch for more context.

Also, I'd like to add that this is my first approach to contribute to this amazing project. And I'm sure I need to do many rounds before this is even close to acceptable but I'd like to give it a go.

My WIP branch: https://github.com/ThorstenBux/three.js/tree/feat/wall-placement-usdz

### PR DESCRIPTION
**Pull request by ThorstenBux**
Related issue: Fixed #21995

**Description**

Until these changes, it wasn't possible to tell AR Quicklook that a USDZ file should be attached to a wall target as opposed to a horizontal surface. 

Reference: https://developer.apple.com/documentation/arkit/usdz_schemas_for_ar/preliminary_anchoringapi

This contribution is funded by [Tripod Digital](https://tripod-digital.co.nz).

### CODE REVIEW

File: examples/jsm/exporters/USDZExporter.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import * as fflate from '../libs/fflate.module.js';
 
 class USDZExporter {
 
-	async parse( scene ) {
+  async parse(scene, options = { ar: { anchoring: { type: "plane"}, planeAnchoring: {alignment: "vertical"} } ) {
```
Comment (mrdoob) [inline examples/jsm/exporters/USDZExporter.js:5]: I would add a `setOptions()` method instead fo the exporter.

**Comment by mrdoob** [comment]
I guess you can remove these `console.log()`s?
Also, make sure you use tabs for indentation.

**Comment by ThorstenBux** [comment]
Thank you @mrdoob for your feedback. I've changed the code accordingly.

**Comment by Mugen87** [comment]
It seems there is still a merge conflict.

**Comment by ThorstenBux** [comment]
Hm, let me see

**Comment by mrdoob** [comment]
Hmm, seems like you changed the indentation to spaces.

**Comment by ThorstenBux** [comment]
arg, VS code auto indentation. I'm used to spaces

**Comment by mrdoob** [comment]
There are still some indentation changes left.

Have a look at the diff:
https://github.com/mrdoob/three.js/pull/21997/files

**Comment by mrdoob** [comment]
Reference: https://developer.apple.com/documentation/arkit/usdz_schemas_for_ar/preliminary_anchoringapi

**Comment by Mugen87** [comment]
Closing in favor of #22854.

**Pull request closed without merging**